src/components/DatePicker/README.md

```diff
@@ -102,3 +102,11 @@ class DatePickerExample extends React.Component {
 ![Date picker on iOS](components/DatePicker/ios/default.png)
 
 <!-- /content-for -->
+
+<!-- content-for: web -->
+
+#### Accessibility
```

Comment:
The Date picker could use the same keyboard interaction treatment, but I also want to be conscientious of how we call out known issues with a component. How detailed should we get about behavior or functionality that should be present but isn't, and has issues logged?

For this example I chose to focus on larger context around fitting the component into the functionality of the page.

Reply:
I'll give this some thought and get back to you

Reply:
> Maybe the answer is some sort of boilerplate caveats section that links to a master issue, and is just part of every accessibility section.

I like the idea of having a boilerplate caveat in general, since the accessibility status of a given implementation might change for an individual user based on a software update, and there will always need to be some unknowns.

Linking to the ARIA spec as the default, and going into further detail if we deviate from it, also helps avoid the case where we need to tweak documentation more frequently.